lib/rules/selector-descendant-combinator-no-non-space/index.js

```diff
@@ -52,6 +51,7 @@ function rule(expectation, options, context) {
 					) {
 						if (context.fix && /^\s+$/.test(value)) {
 							hasFixed = true;
+							// @ts-expect-error -- TS2339: Property 'raws' does not exist on type 'Combinator'.
 							combinatorNode.raws.value = ' ';
```

Comment:
[note] This error is probably caused by the insufficient type definition of `postcss-selector-parser`:

```
lib/rules/selector-descendant-combinator-no-non-space/index.js:55:23 - error TS2339: Property 'raws' does not exist on type 'Combinator'.
```

Reply:
I've opened https://github.com/postcss/postcss-selector-parser/issues/253

Reply:
I've bump up `postcss-selector-parser` and fixed the type error via b95b60db.
The new version includes only my patch. 😃 

- https://github.com/postcss/postcss-selector-parser/releases/tag/v6.0.9
- https://github.com/postcss/postcss-selector-parser/compare/v6.0.8...v6.0.9

(thanks to @alexander-akait 👍🏼 )

Reply:
@jeddy3 Sorry, could you please re-review the change when you have time?